### ISSUE TITLE
Max and select all in one query

### ISSUE THREAD

**Issue by darkovski88** [Raizlabs/DBFlow#358]
Hello,
I need to execute a query like this:
```
select MAX(p.savings_amount ), * from Plan p 
  left join MobileOperator o on o.id_ = p.operator_id 
  left join Handset h on h.planid = p.id_
  group by o.name 
  order by p.savings_amount DESC
```

Im doing this in code by writing:
``` new Select().all().method("MAX", Plan$Table.SAVINGS_AMOUNT).from(Plan.class).as("p")...```

but the query executes the second method called from `all()` and `method("MAX", Plan$Table.SAVINGS_AMOUNT)` so the executed query looks like this:
<code>SELECT MAX(`savings_amount`)  FROM `Plan` AS p LEFT JOIN `MobileOperator` AS o ON `p....</code>. So can anyone tell me how can I execute these two together? Is that possible?

**Comment by agrosner** [comment]
good question @darkovski88  and no its not currently supported. This is a more complex problem that will be fixed 3.0.

**Comment by agrosner** [comment]
Fixed in 3.0, just use `SQLite.select(Method.max(..), Property.ALL_PROPERTY).from(...)..`
